index.js

```diff
@@ -762,6 +779,42 @@ class GithubScm extends Scm {
             sha: pullRequestInfo.head.sha
         }));
     }
+
+    /**
+     * Get an array of scm context (e.g. github:github.com)
+     * @method getScmContexts
+     * @return {Array}
+     */
+    _getScmContexts() {
+        const contextName = this.config.gheHost
+            ? [`github:${this.config.gheHost}`]
+            : ['github:github.com'];
+
+        return contextName;
+    }
+
+    /**
+     * Determine if a scm module can handle the received webhook
+     * @method canHandleWebhook
+     * @param {Object}    headers    The request headers associated with the webhook payload
+     * @param {Object}    payload    The webhook payload received from the SCM service
+     * @return {Promise}
+     */
+    _canHandleWebhook(headers, payload) {
+        return this._parseHook(headers, payload)
+            .then((result) => {
+                if (result === null) {
+                    return Promise.resolve(false);
+                }
+                const checkoutSshHost = this.config.gheHost
+                    ? `git@${this.config.gheHost}:`
+                    : 'git@github.com:';
+
+                return Promise.resolve(result.checkoutUrl.startsWith(checkoutSshHost));
```

Comment:
same here. Can this just be `return result.checkoutUrl.startsWith(checkoutSshHost)`